Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,25 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250])
+    var
+        ConstUpgradeTags: Record "Upgrade Tags";
+    begin
+        ConstUpgradeTags.Reset;
+        ConstUpgradeTags.SetRange(Tag, ExistingTag);
```

Comment:
I do not like this, it should operate in the context of the company. Otherwise we are risking that it is marking too many tags as skipped . Thus it should call the other method with CompanyName() so it is running in the given company, the existing logic behaves like this so it is good to align.

Reply:
Added the companyName() and call the other function